stories/busy-indicator/busy-indicator.stories.js

```diff
@@ -68,6 +69,37 @@ export const contrastMode = () => `<div style="display:flex;justify-content:cent
 contrastMode.parameters = {
     docs: {
         iframeHeight: 200,
-        storyDescription: 'The busy indicator also comes in contrast mode and displays white dots against a dark background. To apply contrast mode, add <code>contrast</code> into the element i.e. <code>fd-busy-indicator--m contrast</code>.'
+        storyDescription:
+            'The busy indicator also comes in contrast mode and displays white dots against a dark background. To apply contrast mode, add <code>contrast</code> into the element i.e. <code>fd-busy-indicator--m contrast</code>.'
+    }
+};
+export const LabelledIndicator = () => `<div style="text-align: center">
+    <div class="fd-busy-indicator fd-busy-indicator--l" aria-hidden="false" aria-label="Loading">
+        <div class="fd-busy-indicator--circle-0"></div>
+        <div class="fd-busy-indicator--circle-1"></div>
+        <div class="fd-busy-indicator--circle-2"></div>
+		<span class="fd-busy-indicator--label">loading data...</span>
+    </div>`;
+
+LabelledIndicator.parameters = {
+    docs: {
+        iframeHeight: 200,
+        storyDescription:
+            'The standard busy indicator animates a sequence of cascading dots expanding and shrinking in a loop. The component comes with a label/message to be displayed when the data is being loaded. <code>fd-busy-indicator--label</code> is used to display the message.'
+    }
+};
+export const ExtendedIndicator = () => `<div style="text-align: center">
+    <div class="fd-busy-indicator--l fd-busy-indicator--extended-indicator fd-padding" aria-hidden="false" aria-label="Loading">
+        <div class="fd-busy-indicator--circle-0"></div>
+        <div class="fd-busy-indicator--circle-1"></div>
+        <div class="fd-busy-indicator--circle-2"></div>
+		<span class="fd-busy-indicator--label">loading data...</span>
+    </div>`;
+
+ExtendedIndicator.parameters = {
+    docs: {
+        iframeHeight: 200,
+        storyDescription:
+            'The standard busy indicator animates a sequence of cascading dots expanding and shrinking in a loop. The component comes with a dialog and a label to be displayed when the data is being loaded. <code>fd-busy-indicator--extended-indicator</code> is used to wrap the busyindiactor in a dialog.'
```

Comment:
```suggestion
            'The standard busy indicator animates a sequence of cascading dots expanding and shrinking in a loop. The component comes with a dialog and a label to be displayed when the data is being loaded. <code>fd-busy-indicator--extended-indicator</code> is used to wrap the busy indicator in a dialog.'
```
Also, just thinking about the modifier names. We have `--label` and `--extended-indicator`; instead how does `--labelled` and `--extended` sound? Just a suggestion.
Actually,  what is the difference between the label and extended indicator? Spec only shows extended indicator. The one within dialog is a `busy dialog`, which has a button additionally, so I don't think this component should have these lines- `...comes with a dialog...` 
![image](https://user-images.githubusercontent.com/53509521/118426173-5afb0400-b6e8-11eb-87a3-bc76763ba7ac.png)

Reply:
the difference is the action/ link present in the dialog box, above screen shot is already added in the dialogue.

the current example was added on the suggestion from @droshev 
https://github.com/SAP/fundamental-styles/pull/2339#pullrequestreview-655087143

Reply:
I understand the difference between extended indicator and busy dialog. I was wondering what the difference is between `__label` example and `extended indicator` example. 
Also, `extended indicator` is not using a dialog, so the description should also not be mentioning anything about a dialog, right?